src/main.cpp

```diff
@@ -136,5 +145,8 @@ int main(int argc, char *argv[])
         std::copy(s.begin(), s.end(), std::back_inserter(bytecode));
     }
 
-    exec(bytecode, argc-a, argv+a);
+    exec(bytecode, debug, argc-a, argv+a);
+    if (debug != nullptr) {
+        delete debug;
```

Comment:
Do not bother to `delete` this. The program is exiting anyway. See https://programmers.stackexchange.com/questions/140483/is-it-a-waste-of-time-to-free-resources-before-i-exit-a-process and http://blogs.msdn.com/b/oldnewthing/archive/2012/01/05/10253268.aspx